res/openvpn-gui-res-fa.rc

```diff
@@ -455,97 +456,95 @@ Options to override registry settings:\n\
 \t\t\t Must be in the range 1 to 61000. Maximum number of configs is limited by 65536 minus this value. Default=25340.\n"
 
 
-    IDS_NFO_USAGECAPTION "OpenVPN GUI Usage"
-    IDS_ERR_BAD_PARAMETER "I'm trying to parse ""%ls"" as an --option parameter \
-but I don't see a leading '--'"
+    IDS_NFO_USAGECAPTION "استفاده از رابط کاربری گرافیکی OpenVPN"
+    IDS_ERR_BAD_PARAMETER "من سعی می کنم "%ls"" را به عنوان یک پارامتر --option تجزیه کنم \
```

Comment:
missing quote --> ""%ls""

Reply:
I will test it, most of the errors are about combining two languages' words this error happened because I use VScode and its LTR, and editing in it is so difficult, I will find an alternative method. 
I don't have enough experience in Make with source code, can you share a summary of it? (parameters)

Reply:
> I will test it, most of the errors are about combining two languages' words this error happened because I use VScode and its LTR, and editing in it is so difficult, I will find an alternative method. I don't have enough experience in Make with source code, can you share a summary of it? (parameters)

I understand, some "smart" editors may automatically re-arrange neutral characters quotes in mixed language, making it difficult to get things like ""xxx"" right.

Anyway, if you push a fixup commit here, we can build using CI and you can then test the exe from the artifacts. That avoids having to setup a local build env.